### PR TITLE
ocm: add search collector

### PR DESCRIPTION
**Pull request by mqhackett**

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/search-collector/open-cluster-management-search-collector-main-postsubmits.yaml
Comment (jlpadilla) [inline ci-operator/jobs/open-cluster-management/search-collector/open-cluster-management-search-collector-main-postsubmits.yaml:56]: Remove empty line.
```suggestion
```

**Review by alvaroaleman — APPROVED** [review]
/label tide/merge-method-squash
/hold

unhold at your convenience

File: ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml
Comment (jlpadilla) [inline ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml:17]: Requires this PR: https://github.com/open-cluster-management/search-collector/pull/108

```suggestion
      memory: 200Mi
      - artifact_dir: /tmp/artifacts
      as: unit
      commands: |
        export SELF="make -f Makefile.prow"
        make -f Makefile.prow unit-tests
      container:
        from: bin
```

File: ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml:8]: Here you're telling the images builder to use the `base` image, but you don't have a `base_images` stanza.

Need to add this section:
```
base_images:
  base:
    name: ubi-minimal
    namespace: ocp
    tag: "8"
```

I also just noticed that this section was deleted from the onboarding docs. I'll get it added back.

Comment (mqhackett) [reply ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml:8]: Thanks for pointing this out!

**Comment by mqhackett** [comment]
Link to spike story: https://github.com/open-cluster-management/backlog/issues/9582
We can use this PR as a reference while onboarding the other projects.

**Comment by jlpadilla** [comment]
@ray-harris We think this is ready to merge, can you take another look and approve?

**Comment by mqhackett** [comment]
@alvaroaleman , Hi! We added some new commits to this PR after your LGTM. Would you mind giving this a look again, and adding the /lgtm so we can merge?

**Comment by alvaroaleman** [comment]
You don't need me for lgtm, anyone who is openshift org member can do that. You only need me to approve but the approve stays :)
/lgtm
/hold cancel

**Comment by mqhackett** [comment]
Ah okay, makes sense. This is our first migration, so thanks for the info!

**Pull request merged**
